tests/e2e/environment-variables/assert-env-vars.yaml

```diff
@@ -0,0 +1,105 @@
+apiVersion: odigos.io/v1alpha1
+kind: InstrumentationConfig
+metadata:
+  name: deployment-nodejs-dockerfile-env
+  namespace: default
+  ownerReferences:
+    - apiVersion: apps/v1
+      blockOwnerDeletion: true
+      controller: true
+      kind: Deployment
+      name: nodejs-dockerfile-env
+status:
+  runtimeDetailsByContainer:
+    - containerName: nodejs-dockerfile-env
+      envFromContainerRuntime:
+        - name: NODE_OPTIONS
+          value: "--require /app/execute_before.js --max-old-space-size=256"
+      language: javascript
+      runtimeVersion: 20.17.0
+      secureExecutionMode: false
```

Comment:
I think these does not belong here. It's adding noise and unrelated checks for a test that verifies something else